app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -9,6 +9,11 @@ class HomeFragment private constructor() : WolmoFragment<FragmentHomeBinding, Ho
     override fun layout() = R.layout.fragment_home
 
     override fun init() {
+        presenter.onViewAttached()
+    }
```

Comment:
You don't need to call `onViewAttached` from here. `onViewAttached` is triggered every time the fragment is instantiated and attached to the presenter.